vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -137,6 +136,23 @@
         </Property>
       </ComplexType>
 
+      <ComplexType Name="Period">
+        <Annotation Term="Core.Description" String="A period to be changed with an action defined in this vocabulary" />
+        <Annotation Term="Core.LongDescription">
+          <String>The action parameter with this type MUST be provided if the action is invoked on a collection
+          without visible data history. It MUST NOT be provided otherwise.
+          Either both `TimeSliceStart` and `TimeSliceEnd` are of type `Edm.Date`
+          or both are of type `Edm.DateTimeOffset`,
+          depending on the `ApplicationTimeSupport/UnitOfTime` of the collection on which the action is invoked.</String>
+        </Annotation>
+        <Property Name="TimeSliceStart" Type="Edm.PrimitiveType" Nullable="false">
+          <Annotation Term="Core.Description" String="Lower boundary of a time slice" />
+        </Property>
+        <Property Name="TimeSliceEnd" Type="Edm.PrimitiveType">

```

Comment:
```suggestion
        <Property Name="PeriodEnd" Type="Edm.PrimitiveType" Nullable="true">
```
To avoid later questions on whether this was intentional.